spec/spec_helper.rb

```diff
@@ -2,8 +2,11 @@
 
 require 'rubocop'
 
-rubocop_gem_path = Gem::Specification.find_by_name('rubocop').gem_dir
-Dir["#{rubocop_gem_path}/spec/support/**/*.rb"].each { |f| require f }
+rubocop_path = File.join(File.dirname(__FILE__), '../vendor/rubocop')
+unless File.directory?(rubocop_path)
+  fail "Can't run specs without a local RuboCop checkout. Look in the README."
```

Comment:
True. I was hoping to get feedback on the code before writing README (and PR description).

> On 22/06/2015, at 23.44, Andy Waite <notifications@github.com> wrote:
> 
> In spec/spec_helper.rb:
> 
> > @@ -2,8 +2,11 @@
> >  
> >  require 'rubocop'
> >  
> > -rubocop_gem_path = Gem::Specification.find_by_name('rubocop').gem_dir
> > -Dir["#{rubocop_gem_path}/spec/support/**/*.rb"].each { |f| require f }
> > +rubocop_path = File.join(File.dirname(__FILE__), '../vendor/rubocop')
> > +unless File.directory?(rubocop_path)
> > +  fail "Can't run specs without a local RuboCop checkout. Look in the README."
> There doesn't seem to be any README in this PR?
> 
> —
> Reply to this email directly or view it on GitHub.
>